ScnSoft-Education/LeetCodeTraining.Tests/RemoveNthNodeFromEndOfList.Tests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using LeetCodeTraining.RemoveNthNodeFromEndOfList18;
+
+namespace LeetCodeTraining.Tests.RemoveNthNodeFromEndOfList
+{
+    public class RemoveNthNodeFromEndOfListTests
+    {
+        [Theory]
+        [InlineData(new int[] { 1, 2, 3, 4, 5 }, 2, new int[] { 1, 2, 3, 5 })]
+        [InlineData(new int[] { 1, 2, 3, 4, 5 }, 1, new int[] { 1, 2, 3, 4 })]
```

Comment:
add unit test where you want to delete elements wich doesn't exist in your array e.g
[InlineData(new int[] { 1, 2, 3, 4, 5 }, 6, new int[] { 1, 2, 3, 4, 5 })]

Reply:
adding test